src/version/sagas/selectedFeedsSagas.js

```diff
@@ -0,0 +1,13 @@
+import { takeEvery, put } from 'redux-saga/effects';
+import { get } from "lodash";
+
+import { SET_SELECTED_FEEDS_ACTION, setSelectedFeedsAction } from "../actions/setSelectedFeedsAction";
+
+export default takeEvery(SET_SELECTED_FEEDS_ACTION.REQUEST, function*(action) {
+    const feeds = get(action, 'data', null);
```

Comment:
I don't understand why you need a saga to do a simple lookup. This could easily be done in the action creator, and therefors you don't need 3 actions (request, success, error) but only one and no saga.

Reply:
I removed sagas and solve this issue only via action and reducer.